runners/spark/src/test/java/org/apache/beam/runners/spark/translation/streaming/utils/SparkTestPipelineOptionsForStreaming.java

```diff
@@ -28,12 +28,6 @@
  */
 public class SparkTestPipelineOptionsForStreaming extends SparkTestPipelineOptions {
 
-  @Override
```

Comment:
👍 this can actually go is the `isStreaming()` option will be automagically set 😄 
Please check if other unit tests are explicitly setting streaming, thanks!